changelogVersions.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Do we want to keep this file in root folder?
Asking out of curiosity 😉

Reply:
I have no idea of a better place 😄 I'm open to suggestions 😉

Reply:
Isn't this more o less connected with `packages/expo` package? 🤔 What would you say if we move it there?

Reply:
If it would be connected then I'd probably use `bundledNativeModules` and take base version from its version ranges. However, I didn't want to clutter it as we need to know which versions we used in the past (actually only from previous SDK) but `bundledNativeModules` gets updated as part of publish command.
Also, why `expo@38.0.0` should know which versions where used in `37.0.0`?